aws-bundle/NOTICE

```diff
@@ -4,3 +4,885 @@ Copyright 2017-2023 The Apache Software Foundation
 
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
+
+--------------------------------------------------------------------------------
+
+NOTICE for Group: commons-codec  Name: commons-codec  Version: 1.15
+
+Apache Commons Codec
+Copyright 2002-2020 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (https://www.apache.org/).
+
+src/test/org/apache/commons/codec/language/DoubleMetaphoneTest.java
+contains test data from http://aspell.net/test/orig/batch0.tab.
+Copyright (C) 2002 Kevin Atkinson (kevina@gnu.org)
+
+===============================================================================
+
+The content of package org.apache.commons.codec.language.bm has been translated
+from the original php source code available at http://stevemorse.org/phoneticinfo.htm
+with permission from the original authors.
+Original source copyright:
+Copyright (c) 2008 Alexander Beider & Stephen P. Morse.
+
+--------------------------------------------------------------------------------
+
+NOTICE for Group: commons-logging  Name: commons-logging  Version: 1.2
+
+Apache Commons Logging
+Copyright 2003-2014 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+--------------------------------------------------------------------------------
+
+NOTICE for Group: org.apache.httpcomponents  Name: httpclient  Version: 4.5.13
+
+Apache HttpClient
+Copyright 1999-2020 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+--------------------------------------------------------------------------------
+
+NOTICE for Group: org.apache.httpcomponents  Name: httpcore  Version: 4.4.13
+
+Apache HttpCore
+Copyright 2005-2020 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+--------------------------------------------------------------------------------
+
+NOTICE for Group: software.amazon.awssdk  Name: annotations  Version: 2.20.18
```

Comment:
Can we deduplicate these and have multiple lines like this for NOTICE blocks that are identical?

Reply:
Sure, this is done.